base/src/base_layers.ts

```diff
@@ -43,6 +43,15 @@ export function nolabels_layers(source: string, c: Theme): any[] {
         "fill-color": c.hospital,
       },
     },
+    {
```

Comment:
Water was covering up pier landuse polygons and buildings on those piers.

After:
<img width="360" alt="image" src="https://github.com/protomaps/basemaps/assets/853051/c3c8c4cb-5b4a-478c-829e-72aeb23390f9">

Before:
<img width="345" alt="image" src="https://github.com/protomaps/basemaps/assets/853051/a153f3ce-cee2-4fbe-91e7-d7f5ce68e5bd">

Reply:
LGTM